sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -90,6 +96,17 @@ public EditContactSteps(
                   "descriptionOfHowContactTookPlace"));
         });
 
+    When(
```

Comment:
This method can be removed, instead collection of UUID can be implemented in collectContactDataDE() with rest of Contact attributes on Edit Contact Page